docs/SRS-Volere/SRS.tex

```diff
@@ -183,13 +183,87 @@ \subsection{Assumptions}
 \end{itemize}
 \section{The Scope of the Work}
 \subsection{The Current Situation}
-\lips
+The process currently employed by the stakeholder is predominantly a manual one.
+Currently, contractor management is a time-consuming and manual effort. The 
+following list identifies the current situation faced by the City:
+\begin{itemize}
+\item To distribute new or revised documentation to contractors, the City sends 
+the documents manually through email. There are no automated processes in place
+and this work flow is done manually for each document. There are dozens of
+contractors which these documents need to be distributed to, and those
+contractors themselves also change over time. The list of these contractors is
+also managed manually.
+
+\item Verification of work performed by contractors currently requires
+a physical visit to site. There is no automated process to collect information 
+such as photos of the work performed, other than manually through email.
+\item It is currently difficult to know whether the labour cost
+billed by the contractors on their invoices is reflective of the actual work
+performed, as that information is only recorded in a physical logbook at each
+site and is not electronic. A contractor could overcharge for the work and it
+would be difficult to prove that their bill is inaccurate.
+
+\end{itemize}
+
 \subsection{The Context of the Work}
-\lips
+The below figure illustrates a high-level context model of the adjacent systems:
+\begin{figure}[h]
+\centering
+\includegraphics[width=1\textwidth]{4G06A-context-model.png}
+\caption{Context model}
+\end{figure}
+
 \subsection{Work Partitioning}
-\lips
+\begin{tabular}{|l|l|}
+\hline
+\textbf{Event Name} & \textbf{Input and Output} \\
+\hline
+1. Contractor Login & Contractor credentials (in) \\
+\hline
+2. Contractor Authentication & GPS location (in) \\
```

Comment:
nitpick, but could we specify that this is location authentication or location verification? Everytime I see us use the word authentication for this I get confused because I think we're letting anyone in the general area of a site log in to our system

Reply:
modified it to Contractor Location Verification.